src/test/java/com/echobox/api/linkedin/types/statistics/StatisticsTest.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.echobox.api.linkedin.types.statistics;
+
+import com.echobox.api.linkedin.jsonmapper.DefaultJsonMapper;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+
+/**
+ * @author clementcaylux on 03/12/2018.
+ */
+public class StatisticsTest {
+
+  /**
+   * test that an statistics JSON is correctly deserialized
+   * json taken from : https://developer.linkedin.com/docs/guide/v2/organizations/
+   * follower-statistics
+   * 
+   */
+  @Test
+  public void testEngagementJson() {
+    
+    String json = "{\n" + "  \"paging\": {\n" + "    \"count\": 10,\n" + "    \"start\": 0\n"
```

Comment:
I think is better to export in a json file and import it like in `DefaultJsonMapperTest`